src/AllThemeResources.h

```diff
@@ -229,11 +228,16 @@ from there.  Audacity will look for a file called "Pause.png".
    DEFINE_IMAGE( bmpBottomFrequencyCursor, wxImage( 32, 32 ), wxT("BottomFrequencyCursor"));
    DEFINE_IMAGE( bmpTopFrequencyCursor, wxImage( 32, 32 ), wxT("TopFrequencyCursor"));
    DEFINE_IMAGE( bmpBandWidthCursor, wxImage(32, 32), wxT("BandWidthCursor"));
-   DEFINE_IMAGE( bmpSubViewsCursor, wxImage(32, 32), wxT("SubViewsCursor"));
 
    //SET_THEME_FLAGS(  resFlagNewLine  );
 
+// DA: The logo with name xpm has a different width.
+#ifdef EXPERIMENTAL_DA
```

Comment:
Why did you add EXPERIMENTAL_DA stuff? We're trying to remove anything related to DA so this doesn't help.

Reply:
Sorry. Feel free to come up with something better.

Reply:
I don't want to be rude but you can simply update the file and removing the stuff, y'know? ;)

Reply:
Ah, my bad. You should keep only the line `#define LOGOWITHNAME_WIDTH 506`.